roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -36,6 +36,12 @@
     state: present
   when: not openshift.common.is_atomic | bool
 
+- name: Check we are not using node as a Docker system container with CRI-O
```

Comment:
Shouldn't this be for Docker container with the system container being OK?

Reply:
ouch, yes!   Pushed a new version